.github/workflows/add-issue-to-project.yml

```diff
@@ -1,46 +1,14 @@
 name: Add issue to project
 on:
   issues:
-    types:
-      - opened
+    types: [opened, transferred]
+
 jobs:
-  track_issue:
+  add-to-project:
+    name: Add issue to project
     runs-on: ubuntu-latest
     steps:
-      - name: Get project data
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_PROJECT_ACCESS }}
-          ORGANIZATION: ${{ github.repository_owner }}
-          PROJECT_NUMBER: 4
-        run: |
-          gh api graphql -f query='
-            query($org: String!, $number: Int!) {
-              organization(login: $org){
-                projectNext(number: $number) {
-                  id
-                  fields(first:20) {
-                    nodes {
-                      id
-                      name
-                      settings
-                    }
-                  }
-                }
-              }
-            }' -f org=$ORGANIZATION -F number=$PROJECT_NUMBER > project_data.json
-
-          echo 'PROJECT_ID='$(jq '.data.organization.projectNext.id' project_data.json) >> $GITHUB_ENV
-                    
-      - name: Add issue to project
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_PROJECT_ACCESS }}
-          ISSUE_ID: ${{ github.event.issue.node_id }}
-        run: |
-          item_id="$( gh api graphql -f query='
-            mutation($project:ID!, $issue:ID!) {
-              addProjectNextItem(input: {projectId: $project, contentId: $issue}) {
-                projectNextItem {
-                  id
-                }
-              }
-            }' -f project=$PROJECT_ID -f issue=$ISSUE_ID --jq '.data.addProjectNextItem.projectNextItem.id')"    
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/AbsaOSS/projects/4
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
It doesn't really matter, but just for my curiosity - can't `secrets.GH_TOKEN_PROJECT_ACCESS` be reused?